lib/log.js

```diff
@@ -66,17 +66,22 @@ Lug.prototype.event = function (name, data) {
   process.stdout.write(JSON.stringify(e) + '\n')
 }
 
-Lug.prototype.activityEvent = function(event, uid, request) {
+Lug.prototype.activityEvent = function(event, request, data) {
   var info = {
-    event: event,
-    uid: uid,
```

Comment:
Hmmm. Throwing feels wrong to me, I don't think a logging function should be responsible for failing a request which has otherwise completed successfully. But there aren't any tests that `activityEvent` is called and writing them is not straightforward either.

Would a comment be sufficient for now? If not, maybe it should revert to being a listed argument, as before?